app/models.py

```diff
@@ -24,7 +22,7 @@ def find_by_name(self):
 
 
 
-class User(db.Model):
+class User(UserMixin, db.Model):
```

Comment:
Why don't you use the base model here? You have 2 times the id field.
Is better to have

```
class User(BaseModel):
    # without id field
```